src/lib.rs

```diff
@@ -56,20 +64,34 @@ macro_rules! __winit_test_internal_collect_test {
 #[doc(hidden)]
 // This part is semver-exempt.
 pub mod __private {
-    use winit::event_loop::EventLoopBuilder;
     pub use winit::event_loop::EventLoopWindowTarget;
+    use winit::event_loop::{ControlFlow, EventLoopBuilder};
 
     use owo_colors::OwoColorize;
+    use std::any::Any;
     use std::panic::{catch_unwind, AssertUnwindSafe};
+    #[cfg(not(target_arch = "wasm32"))]
     use std::time::Instant;
+    #[cfg(target_arch = "wasm32")]
+    use web_time::Instant;
```

Comment:
I could also just use `web-time`, it exports `std::time` when not using the `wasm32-unknown-unknown` target.